vunit/sim_if/ghdl.py

```diff
@@ -375,8 +381,14 @@ def simulate(  # pylint: disable=too-many-locals
             gcov_env["GCOV_PREFIX"] = coverage_dir
             self._coverage_test_dirs.add(coverage_dir)
 
+        if precompiled:
+            precompiled_binary_path = Path(precompiled)
+            cwd = precompiled_binary_path.parent.absolute()
```

Comment:
Since precompiled_binary_path is used once only, I think it's reasonable to do:

`cwd = precompiled.parent.absolute() if precompiled else None`

Reply:
I'll look into it

Reply:
Done